app.sh

```diff
@@ -32,4 +32,6 @@ elif [ "$THOTH_WORKFLOW_TASK" = "update_kebechet_installation" ]; then
     exec python3 update_keb_installation.py
 elif [ "$THOTH_WORKFLOW_TASK" = "create_purge_issues" ]; then
     exec python3 create_purge_issues.py
+elif [ "$THOTH_WORKFLOW_TASK" = "thoth_advise_demo" ]; then
```

Comment:
This work could be promoted to a new feature of Thoth where a user can post an URL to GitHub repo and the workflow on the backend will setup the repository so that it is compatible with Thoth with additional guidance and such. In that case, "demo" could be substituted with something related to git/github/repository/... The feature does not need to be used solely be project Meteor.

Reply:
How do you feel about `thoth_repository_initialization`?

Reply:
sounds good 👍🏻

Reply:
> sounds good 👍🏻

good 2 go